sys/mips/mips/pmap.c

```diff
@@ -2728,6 +2780,10 @@ void
 pmap_copy_page_tags(vm_page_t src, vm_page_t dst)
 {
 
+	// XXX CHERI capdirty: should we assert that dst is CAPSTORE or
```

Comment:
I guess we probably want to convince ourselves either way for this? But if not, this should be a C-style comment.